src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx

```diff
@@ -16,36 +16,86 @@ redirects:
   - /docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-best-practices
 ---
 
-This document describes some best practices and examples to help you get the most out of your alerts configuration.
+Improve your alert coverage by implementing the following recommendations. 
+
+Read on to learn the best practices for:
+- [Policies](/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices/#policy-practices)
+- [Notification channels](/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices/#channel-practices)
+- [Incident preferences](/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices/#incident-practices)
+- [Thresholds and violations](/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices/#threshold-practices)
+- [Muting roles](/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices/#mute-practices)
 
 <Callout variant="important">
-  Before reading this, we recommend you read [Alerts concepts and workflow](/docs/alerts/new-relic-alerts/getting-started/alert-policy-workflow).
+  Before reading this, we recommend you also read [Alerts concepts and workflow](/docs/alerts/new-relic-alerts/getting-started/alert-policy-workflow).
 </Callout>
 
-## Define policies for entities or people [#entities-people]
+## Recommended alerts [#recommend-alerts]
+
+Use [recommended practices](https://discuss.newrelic.com/t/announcing-alert-recommendations-for-apm-applications/154542) if you are new to Alerts or if you want suggestions that optimize your alert coverage.
+
+## Policies [#policy-practices]
+
+A policy is a container for alike conditions.
+
+<Callout variant="tip">
+Learn how to [create, edit, or find policies](/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/create-edit-or-find-alert-policy/) if you’re new to Alerts.
+</Callout>
 
-When designing your policies, consider:
+Organize your policy by surrounding its conditions to a single entity and to the specific team that needs to be notified when a problem arises within that entity.
```

Comment:
Watch your prepositions. In "conditions to a single entity", for example, "to" feels odd there. Especially related to the word "surrounding". Definitely check that in the rest of the doc too.

I do really like the impulse behind this rewrite, though.

Reply:
Think about how someone might respond to this sentence if they're unfamiliar with any of these terms: "policy", "condition", "entity".

Reply:
In general, throughout the alerts docs, there's a lot of consideration for the technical detail, but... a lot less consideration for the reader. Our ongoing task is to think about how we can swap that emphasis and focus: consider the reader, but also provide enough technical detail for these docs to still be meaningful.